### PR TITLE
Amina/weeks_array

### PR DESCRIPTION
**Pull request by amina-deriv**

### CODE REVIEW

File: src/components/month-picker.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+    const handleOnDismiss = () => {
+        setVisibility(false);
+    };
+    const getWeeksInMonth = (year, month) => {
+        const weeks = [];
+        const firstDate = new Date(year, month - 1, 1);
+        const lastDate = new Date(year, month, 0);
+        const numDays = lastDate.getDate();
+        console.log(numDays);
```
Comment (bahar-sadeghzadeh) [inline src/components/month-picker.tsx:49]: Isn't it better to remove this `console`😁

File: src/components/month-picker.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+        while (start <= numDays) {
+            weeks.push({ start: start, end: end });
+            start = end + 3;
+            end = end + 7;
+            end = start === 1 && end === 8 ? 1 : end;
+            if (end > numDays) end = numDays;
+        }
+
+        console.log(weeks)
```
Comment (bahar-sadeghzadeh) [inline src/components/month-picker.tsx:70]: Also this one😁

File: src/components/month-picker.tsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        // return weeks;
```
Comment (bahar-sadeghzadeh) [inline src/components/month-picker.tsx:71]: I think this line commented mistakenly

File: src/pages/index.tsx [hunk 4]
```diff
@@ -1,5 +1,7 @@
 import type { NextPage } from 'next'
+import React, { useState } from "react";
```
Comment (bahar-sadeghzadeh) [inline src/pages/index.tsx:2]: This line can be omitted😄

File: src/components/month-picker.tsx (hunk 1 above)
Comment (amina-deriv) [reply src/components/month-picker.tsx:49]: @Bahar-FS consoled for now since it needs to be passed to the component farrah is working on.

**Pull request merged**